src/test/java/seedu/attendance/AttendanceListTest.java

```diff
@@ -21,6 +21,6 @@ void isEmpty() {
     void getAttendanceList() {
         assertEquals("[]", attendanceList.toString());
         attendanceList.add(new Attendance("John Doe", "Y"));
-        assertEquals("[John Doe Y]", attendanceList.toString());
+        assertEquals("[John Doe Present]", attendanceList.toString());
```

Comment:
I have actually added a `toString()` method for `attendanceList`, so the expected output is `[John Doe: Present]`

Reply:
Okay, I think in that case, u might have to change it at your code because you wrote it in your code. Noted on the change and thanks for doing up the storage!